content/client-server-api/_index.md

```diff
@@ -212,6 +212,29 @@ only read state (e.g.: `/sync`, get account data, etc).
 The user is unable to reject an invite to join the server notices room.
 See the [Server Notices](#server-notices) module for more information.
 
+#### Rate limiting
+
+Homeservers SHOULD implement rate limiting to reduce the risk of being
+overloaded. If a request is refused due to rate limiting, it should
+return a standard error response of the form:
+
+```json
+{
+  "errcode": "M_LIMIT_EXCEEDED",
+  "error": "string",
+  "retry_after_ms": integer (optional, deprecated)
+}
+```
+
+{{% added-in v="1.10" %}}
+
+The [`Retry-After`](https://www.rfc-editor.org/rfc/rfc9110#field.retry-after)
+HTTP header SHOULD be specified by the server on any 429 code response.
```

Comment:
Avoid the passive voice where possible. "Homeservers should include a `Retry-After` header ...".